examples/framesjs-starter/app/debug/page.tsx

```diff
@@ -68,8 +69,9 @@ export default function Page({
         fid: farcasterUser.fid,
         buttonIndex,
         castId,
-        url: Buffer.from(currentFrame.frame.postUrl),
-        inputText: Buffer.from(inputText),
+        url: Buffer.from(url),
+        // seems the message in hubs actually requires a value here.
+        inputText: Buffer.from(inputText ?? ""),
```

Comment:
see https://github.com/framesjs/frames.js/pull/12/files#diff-eab8c631bb8f0e313ebb22527cd6d53bba944a99870ffae6ed02bc13bab2fc2eL34

Reply:
I don't follow

Reply:
so it's fine?

Reply:
yes